crates/nostr-sqldb/migrations/sqlite/2025-04-11-095120_events/up.sql

```diff
@@ -0,0 +1,27 @@
+-- The actual event data
+CREATE TABLE events (
+    id VARCHAR(64) PRIMARY KEY,
+    pubkey VARCHAR(64) NOT NULL,
+    created_at BIGINT NOT NULL,
+    kind BIGINT NOT NULL,
+    payload BLOB NOT NULL,
+    signature VARCHAR(128) NOT NULL,
```

Comment:
Since on query the `payload` is used for deserialize the event, can the `signature` be removed?